cmd/queue/main.go

```diff
@@ -79,6 +79,7 @@ const (
 	responseTimeInMsecN    = "request_latencies"
 	appRequestCountN       = "app_request_count"
 	appResponseTimeInMsecN = "app_request_latencies"
+	queueDepthN            = "app_queue_depth"
```

Comment:
Could you remove the prefix `app`? It is not about the customer app container.

Reply:
Done.